hphp/test/zend/good/ext/standard/tests/strings/strncmp_variation6.php.expectf

```diff
@@ -771,6 +771,6 @@ int(0)
 int(0)
 
 -- Checking with out of character's range, given in binary format --
-int(1)
+bool(true)
 
-*** Done ***
```

Comment:
If we could not add a newline at the end of the file that would be nice, but not a big deal.